engines/scc_proxy/lib/scc_proxy/engine.rb

```diff
@@ -201,14 +201,19 @@ def parse_error(error_message, token = nil, email = nil)
       error_message
     end
 
-    def get_scc_activations(headers, system_token, mode)
+    def get_scc_activations(headers, system_token, mode, login)
```

Comment:
Why not using the `system` as a parameter here instead of it's attributes `system_token`, `mode`, `login`? Same for method `scc_check_subscription_expiration()`.

Reply:
yes, thought about it, it required a bit more refactor that I thought to do in a different PR but
I think it could be done here too